### PR TITLE
Fixed #24736: Document the limit attribute of Sitemap class

### PR DESCRIPTION
**Pull request by abhaga**
Document the limit attribute in Sitemap class which can be overridden to lower the number of urls being included per page.

### CODE REVIEW

File: docs/ref/contrib/sitemaps.txt [hunk 1]
```diff
@@ -232,6 +232,17 @@ Sitemap class reference
...
         context of a request, the default is ``'http'``.
 
+    .. attribute:: Sitemap.limit
+
+        **Optional.**
+
+        This attribute defines the maximum number of urls included per page. Its
+        value should not exceed ``50000`` which is the upper limit defined by 
+        Google. Default value is ``50000``. See the `index documentation on
+        sitemaps.org`_ for more.
```
Comment (MarkusH) [inline docs/ref/contrib/sitemaps.txt]: ... for more information.

File: docs/ref/contrib/sitemaps.txt [hunk 2]
```diff
@@ -232,6 +232,17 @@ Sitemap class reference
         sitemap was requested is used. If the sitemap is built outside the
         context of a request, the default is ``'http'``.
 
+    .. attribute:: Sitemap.limit
+
+        **Optional.**
+
+        This attribute defines the maximum number of urls included per page. Its
+        value should not exceed ``50000`` which is the upper limit defined by
```
Comment (MarkusH) [inline docs/ref/contrib/sitemaps.txt]: Maybe rephrase to "... upper limit allowed/accepted by Google"

**Comment by timgraham** [comment]
merged in 9096e2b5f75abf8e8882937bd3c3d47ccdc24e25, thanks!
